volumes/miovision/sql/function-find_invalid_movements.sql

```diff
@@ -0,0 +1,80 @@
+﻿CREATE OR REPLACE FUNCTION miovision_api.find_invalid_movements(
```

Comment:
Weird this is committed in the `automate_miovision_aggregation` when it doesn't exist in `miovision_new`

Reply:
I think the function got superseded in `miovision_new` with `miovision_new.intersection_movements`.

Reply:
Ah, I moved the old query to `csv_data` and then I'm keeping this, because I think this is what we will be using going forward